src/main/kotlin/no/nav/omsorgspenger/overføringer/statistikk/OverføringerStatistikkService.kt

```diff
@@ -0,0 +1,85 @@
+package no.nav.omsorgspenger.overføringer.statistikk
+
+import no.nav.k9.statistikk.kontrakter.Behandling
+import no.nav.k9.statistikk.kontrakter.Sak
+import org.apache.kafka.clients.producer.KafkaProducer
+import java.time.OffsetDateTime
+
+internal class OverføringerStatistikkService(
+        private val kafkaProducer: KafkaProducer<String, String>,
+        private val topic: String = "privat-k9-TODO-???") { // TODO: TOPIC for sak og behandling
+
+    fun publiser(statistikk: OverføringStatistikkMelding) {
+        val tekniskTid = OffsetDateTime.now()
+
+        val saksnummer = statistikk.saksnummer
+
+        val avsender = "omsorgspenger-rammemeldinger"
+
+        val sak = Sak(
```

Comment:
Var det ikke slik at vi ikke trenger å sende noe egen for sak? Bare sende `Behandling` ?

Reply:
Jeg er litt usikker. Det er noen felt som kun finnes på `Sak`. Det kan hende at de lager et nytt schema, i så fall vil jeg bare mappe opp alt først, for den vanskelige biten av jobben er å finne alle feltene.